### PR TITLE
[SASS-3783] Removed duplicate page.

### PR DESCRIPTION
**Pull request by erasmos-hmrc**
The removed code was introduced by SASS-2654, which was a duplicate of the page introduced by SASS-3233.

# Checklist

Jira subtask: SASS-3833

## Developer: Before Asking Others To Review Your Code

Have you ...

- [X]  Had a QA verify that all acceptance criteria have been met, with comment(s) in the ticket? 
- [X]  Hosted a group review of your implementation?
- [X]  Rebased against `main`? 
- [X]  Ensured that your commit(s) contain the ticket key?
- [X]  Ensured that your branch is named after the ticket key?
- [X]  Successfully run the unit tests?
- [X]  Successfully run the integration tests?
- [x]  Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.
- [x]  Ensured that the PR job has passed.

## Reviewer: Before Reviewing

Have you ...

- [x]  Pulled the branch down?
- [x]  Assigned yourself to this PR?
- [x]  Moved the Jira *subtask* to “in review”?
- [x]  Ensured that it is rebased against `main`?

## Developer: After Approval

Have you ...

- [ ]  Rebased against `main`?
- [ ]  Moved the Jira *subtask* to “in pipeline”?

## Developer: After Merging

Have you ...

- [ ]  Deleted the branch?
- [ ]  Scheduled a demo? If not, please add a comment.

### CODE REVIEW

**Comment by erasmos-hmrc** [comment]
> Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.

@walesoares ran them for me, as I haven't fixed my env yet.

File: conf/messages [hunk 1]
```diff
@@ -24,7 +24,6 @@ common.use = use
 common.error.errorKey = Error:
 common.opensInNewTab = (opens in new tab)
 common.currency.hint = For example, £193.52
-common.currency.hint1 = For example, £193.54
```
Comment (srinivaspach) [inline conf/messages:27]: is this not used by any other page? I remember seeing this hint in someone pages too

**Review by srinivaspach — CHANGES_REQUESTED** [review]

File: conf/messages (hunk 1 above)
Comment (erasmos-hmrc) [reply conf/messages:27]: I confirmed that `hint1` is only used in that page. It's `hint` which is still used.

Comment (srinivaspach) [reply conf/messages:27]: The fact that sbt it:test passed, is enough to prove this.

**Review by srinivaspach — APPROVED** [review]
LGTM

**Comment by erasmos-hmrc** [comment]
> Scheduled a demo? If not, please add a comment.

No need: just a removal of a page that was already demo'd - and still exists elsewhere.

**Pull request merged**